library/BUILD

```diff
@@ -5,3 +5,34 @@ filegroup(
     srcs = ["proguard.txt"],
     visibility = ["//visibility:public"],
 )
+
+genrule(
+    name = "javadocs",
+    srcs = [],
+    outs = ["envoy-javadoc.jar"],
+    cmd = """
+orig_dir=$$PWD
+tmp_dir=$$(mktemp -d)
+java -jar $(location @kotlin_dokka//jar) $$orig_dir/library/java/src/ -format javadoc -output $$tmp_dir
+java -jar $(location @kotlin_dokka//jar) $$orig_dir/library/kotlin/src/ -format javadoc -output $$tmp_dir
```

Comment:
Can you pass multiple directories in one dokka call?

Reply:
yeah